static/new_home/style.css

```diff
@@ -631,6 +631,58 @@ header
   color: #eb5d1e;
 }
 
+@media screen and (min-width: 768px)
+  {
+    .contact-us-grid
+    {
+      display: grid;
+    }
+    .contact-us-bottom-card{
+      margin-left: -10px;
+    }
+  }
+
+  @media screen and (max-width: 767px)
+  {
+    .contact-us-bottom-card
+    {
+      margin-top: -70px;
```

Comment:
You should avoid negative margins as possible. Instead of this, you can remove the bottom margin of the top card and add a negative margin of -10px here in the bottom card.